docs/Helix.md

```diff
@@ -120,3 +120,16 @@ There are two main ways to opt out of helix
 - Skipping an individual test via `[SkipOnHelix("url to github issue")]`.
 
 Make sure to file an issue for any skipped tests and include that in a comment next to either of these
+
+## Process for updating helix matrix
+
+Goal is to balance cost/flakiness against having some coverage of supported distros:
+- At the start of each product version, we pick a set of queues/versions/arches to run based on popularity and perceived risk, and how long is left in the support for that OS version.
+- Whenever a new OS is coming online, we ask CTI to do a run on it, and if there is support for it in helix, we submit a PR to update our helix-matrix to include it for it to check for any failures in it, but if there aren’t any, we don’t merge it.
```

Comment:
Suggest mentioning potential need to update dotnet/dotnet-buildtools-prereqs-docker if Helix support doesn't (yet) exist e.g.
```suggestion
- Whenever a new OS is coming online, we ask CTI to do a run on it, and if there is support for it in helix, we submit a PR to update our helix-matrix to include it for it to check for any failures in it, but if there aren’t any, we don’t merge it.
    - If an appropriate queue does not yet exist, we could submit a PR to https://github.com/dotnet/dotnet-buildtools-prereqs-docker to add it. This helps even if we do not plan to keep the dotnet/aspnetcore change around.
```